src/sbt-test/sbt-autoversion/simple/test

```diff
@@ -0,0 +1,104 @@
+# Generate project files.
+> reload
```

Comment:
I don't think this one is required, since you have no "state" you might need to refresh a that point

Reply:
You're right, I always start with `> reload` out of habit but it isn't necessary, can trim.